src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -40,7 +39,51 @@ static void Main()
         Console.Write(rn);
     }
 }";
-            CompileAndVerify(source: source, expectedOutput: "TwoZero");
+            CompileAndVerify(source: source, expectedOutput: "TwoZero")
+                .VerifyOperationTree(@"
+Test.Main()
```

Comment:
I think we will likely have following category of tests:

1. Inline VerifyOperationTree tests for small method bodies - this will not clutter the existing binding tests as the tree will supposedly be pretty small.
2. VerifyOperationTree tests on large method bodies - I agree that we should put the expected operation tree for such tests into a separate file.
3. Hand crafted tests with SyntaxNode based operation tree verification - similar to the ones already written by John and proposed by @AlekseyTs is another comment. I'll add a separate test framework helper to verify expected Operation tree given a syntax node.